packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -228,6 +238,19 @@ class PropertyEditorController extends DisposableController
     );
   }
 
+  List<EditableProperty> _extractProperties(EditableArgumentsResult? result) =>
+      (result?.args ?? <EditableArgument>[])
+          .map(argToProperty)
+          .nonNulls
+          // Filter out any deprecated properties that aren't set.
+          .where((property) => !property.isDeprecated || property.hasArgument)
+          .toList();
+
+  List<CodeAction> _extractRefactors(CodeActionResult? result) =>
+      (result?.actions ?? <CodeAction>[])
+          .where((action) => action.title != null && action.command != null)
```

Comment:
Have you seen data where any of these were null? In theory, you should only ever be getting `Command` back from `textDocument/codeAction`, so both fields should be non-null (except for bugs). If you get a `CodeActionLiteral` back (you should not, but other editor LSP clients may) then `title` should still always be non-null (per the LSP spec), although `command` could be null (and if not, would be a `Command` rather than a `String`).

Reply:
No I haven't! I added this check just to be safe, though I could also make `title` and `command` non-nullable if we expect them to always be returned. WDYT?